cfme/tests/infrastructure/test_host_hostname.py

```diff
@@ -0,0 +1,23 @@
+from utils import testgen
+from cfme.web_ui import Quadicon, toolbar
+from cfme.infrastructure.host import Host
+from cfme.configure.tasks import is_host_analysis_finished
+from utils.wait import wait_for
+import pytest
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'], scope='module')
+pytestmark = [pytest.mark.usefixtures("setup_provider_modscope")]
+
+
+def test_host_hostname(provider):
```

Comment:
Can you also add a docstring? Like https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cli/test_rake_evm_automate-53.py#L38

The contents of docstrings get uploaded to Polarion once the test runs. And we want it to be on each test.